### PR TITLE
Added release notes 3.4

### PR DESCRIPTION
**Pull request by trusli**
Appendix will be separated in another file

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,6 +1,210 @@
...
+------------------------------------------------------------------------------------------------------------------------
+### Summary
+Affinity rules provides a way to specify hints on how operators should be deployed in a Hadoop cluster. There are two types of rules: affinity and anti-affinity rules. Affinity rule indicates that the group of operators in the rule should be allocated together. Anti-affinity rule, the new feature in Apache Apex 3.4.0, indicates that the group of operators should be allocated separately.
+
+This release also include a lot of bug fixes. Please see appendix for full list. 
+
+### dtManage
+* User can restart a killed application from dtManage
+* "Retrieve Ended Apps" button change to "Hide Ended Apps" after dtManage retrieve killed apps.
+* User can use mouse scrool to zoom in/out of physical DAG view
```
Comment (davidyan74) [inline docs/release_notes.md]: scroll?

File: docs/release_notes.md [hunk 2]
```diff
@@ -1,6 +1,210 @@
 DataTorrent RTS Release Notes
 ========================================================================================================================
 
+Version 3.4.0 
+------------------------------------------------------------------------------------------------------------------------
+### Summary
+Affinity rules provides a way to specify hints on how operators should be deployed in a Hadoop cluster. There are two types of rules: affinity and anti-affinity rules. Affinity rule indicates that the group of operators in the rule should be allocated together. Anti-affinity rule, the new feature in Apache Apex 3.4.0, indicates that the group of operators should be allocated separately.
+
+This release also include a lot of bug fixes. Please see appendix for full list.
```
Comment (davidyan74) [inline docs/release_notes.md]: includes?

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,6 +1,210 @@
...
+Version 3.4.0 
+------------------------------------------------------------------------------------------------------------------------
+### Summary
+Affinity rules provides a way to specify hints on how operators should be deployed in a Hadoop cluster. There are two types of rules: affinity and anti-affinity rules. Affinity rule indicates that the group of operators in the rule should be allocated together. Anti-affinity rule, the new feature in Apache Apex 3.4.0, indicates that the group of operators should be allocated separately.
+
+This release also include a lot of bug fixes. Please see appendix for full list. 
+
+### dtManage
+* User can restart a killed application from dtManage
+* "Retrieve Ended Apps" button change to "Hide Ended Apps" after dtManage retrieve killed apps.
```
Comment (davidyan74) [inline docs/release_notes.md]: grammar problem

**Comment by trusli** [comment]
I made the corrections

**Pull request merged**